Onboarding/Sources/Onboarding/Onboarding.swift

```diff
@@ -1,2 +1,27 @@
 // The Swift Programming Language
 // https://docs.swift.org/swift-book
+
+import Foundation
+import SwiftUI
+import UIKit
+
+public final class Onboarding {
+    private let dismissAction: () -> Void
+    private let onboardingSource: OnboardingSource
+
+    public init(dismissAction: @escaping () -> Void,
+                onboardingSource: OnboardingSource) {
```

Comment:
Can you switch these around, please. Handlers at the end.
Can you also change `onboardingSource:` -> `source:` assuming no naming conflicts or anything

Reply:
done